src/burgermenu.h

```diff
@@ -3,12 +3,73 @@
 #include <QList>
 #include <QSize>
 #include <QIcon>
+#include <QPushButton>
+#include <QPainter>
+#include <QStyleOptionButton>
+#include <QStyle>
 
-class QPushButton;
 class QActionGroup;
 class QAction;
 class QString;
 
+extern const QString BurgerButtonObjectName;
+extern const QString BurgerMenuName;
+extern const QString MainBurgerButtonObjectName;
+
+class BurgerButton : public QPushButton
+{
+    Q_OBJECT

```

Comment:
@chrisaverage Looks like there must be `Q_OBJECT`, then `BurgerButton` must be declared in a header file.